trace-normalization/src/normalizer.rs

```diff
@@ -16,8 +16,18 @@ const YEAR_2000_NANOSEC_TS: i64 = 946684800000000000;
 // DEFAULT_SPAN_NAME is the default name we assign a span if it's missing and we have no reasonable fallback
 const DEFAULT_SPAN_NAME: &str = "unnamed_operation";
 
+const TAG_SAMPLING_PRIORITY: &str = "_sampling_priority_v1";
+const TAG_ORIGIN: &str = "_dd.origin";
+
+#[allow(dead_code)]
+const SAMPLER_PRIORITY_AUTO_DROP: i32 = 0;
 #[allow(dead_code)]
-pub fn normalize(s: &mut pb::Span) -> anyhow::Result<()> {
+const SAMPLER_PRIORITY_AUTO_KEEP: i32 = 1;
+#[allow(dead_code)]
+const SAMPLER_PRIORITY_USER_KEEP: i32 = 2;
+const SAMPLER_PRIORITY_NONE: i32 = i8::MIN as i32;
+
+fn normalize(s: &mut pb::Span) -> anyhow::Result<()> {
```

Comment:
Perhaps this needs a more descriptive name given all the other normalize functions in this file

Reply:
good idea, I'll change it